src/test/java/org/jenkinsci/plugins/workflow/job/MemoryCleanupTest.java

```diff
@@ -24,6 +26,8 @@
 
     @Test
     public void cleanup() throws Exception {
+        // Ignore on Java 9+, because `MemoryAssert.assertGC` below tries to call setAccessible on some internal structure
+        assumeTrue(System.getProperty("java.specification.version").startsWith("1."));
```

Comment:
WDYT about `assumeTrue(JavaSpecificationVersion.forCurrentJVM().isOlderThanOrEqualTo(JavaSpecificationVersion.JAVA_8));`
?

Reply:
This line shouldn't be there actually. I was supposed to remove it in favor of the change directly in JTH.